### ISSUE TITLE
Error using Group Generator

### ISSUE THREAD

**Issue by WaetzoldP** [danielhrisca/asammdf#56]
# Python version
_Please run the following snippet and write the output here_
```python
import platform
import sys
from pprint import pprint

pprint("python=" + sys.version)
pprint("os=" + platform.platform())

try:
    import numpy
    pprint("numpy=" + numpy.__version__)
except ImportError:
    pass

try:
    import asammdf
    pprint("asammdf=" + asammdf.__version__)
except ImportError:
    pass
```

('python=3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 10:22:32) [MSC v.1900 '
 '64 bit (AMD64)]')
'os=Windows-10-10.0.14393-SP0'
'numpy=1.14.1'
'asammdf=3.2.1'

# Code 

  ## Code snippet
from asammdf import MDF

class MultiPandas(object):

    def __init__(self, mdfpath):
        self.__mdf = MDF(mdfpath)
        iterGroups = self.__mdf.iter_groups()
        self.__groups = []
        finished = next(iterGroups)
        self.__groups.append(finished)

			
if __name__ == "__main__":
    filename = ...
    multipandas = MultiPandas(filename)
  
  ## Traceback
  File "D:/Projekte/PythonKI/CoreData/MultiPandas.py", line 48, in <module>
    multipandas = MultiPandas(filename)
  File "D:/Projekte/PythonKI/CoreData/MultiPandas.py", line 23, in __init__
    finished = next(iterGroups)
  File "C:\ProgramData\Anaconda3\envs\osmnx\lib\site-packages\asammdf\mdf.py", line 1222, in iter_groups
    master.append(self.get_master(i, data=data_bytes))
  File "C:\ProgramData\Anaconda3\envs\osmnx\lib\site-packages\asammdf\mdf_v3.py", line 3123, in get_master
    data_bytes, offset = fragment
ValueError: not enough values to unpack (expected 2, got 1)

  
# Description
Error when using MDF.iter_groups()

**Comment by danielhrisca** [comment]
Hello,

please try out the latest code from the branch development.

Also consider using the pandas export (see method _export_ from the documentation http://asammdf.readthedocs.io/en/development/api.html#mdf)

**Comment by WaetzoldP** [comment]
Fine, the original Error is gone, BUT

Is this correct? the data frames only contain time axis:

![29-03-_2018_16-59-20](https://user-images.githubusercontent.com/10012749/38096316-db5e57a2-3372-11e8-9f8c-1adf723c318d.png)

**Comment by danielhrisca** [comment]
You are right. I just had a fast look over it before I had to go. I'll get it properly fixed tonight.

In the mean time you should be able to use the export.

**Comment by danielhrisca** [comment]
I have prepared a new fix that works with the files that I've tested it with.

Let me know if it's good for you too.

**Comment by danielhrisca** [comment]
@WaetzoldP 
I've released version 3.3.0 that contains the fix for this issue. Please give it a try.
